### PR TITLE
rpc|llmq|test: Implement `verifychainlock`

### PR DESCRIPTION
**Pull request by xdustinface**
Implements a wrapper for ChainLock verification similar to  `verifyislock` for InstantSend verification. This has been requested by the platform team.

### CODE REVIEW

**Review by PastaPastaPasta — DISMISSED** [review]
utACK

File: src/rpc/rpcquorums.cpp [hunk 1]
```diff
@@ -655,6 +659,48 @@ UniValue quorum(const JSONRPCRequest& request)
     }
 }
 
+void verifychainlock_help()
+{
+    throw std::runtime_error(
+            "verifychainlock \"blockHash\" \"blockHeight\" \"signature\"\n"
```
Comment (UdjinM6) [inline src/rpc/rpcquorums.cpp:665]:
```suggestion
            "verifychainlock \"blockHash\" \"signature\" ( blockHeight )\n"
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Review by PastaPastaPasta — APPROVED** [review]
utACk

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**